include/aspect/particle/property/interface.h

```diff
@@ -259,7 +259,13 @@ namespace aspect
          * integration of solution properties or time varying particle
          * properties are used while solving the model problem.
          */
-        update_time_step
+        update_time_step,
+        /**
+         * Update the particles at every advection substep. This is only necessary
```

Comment:
Can you explain what a "substep" is? Do you mean "after every solve that updates one of the compositional fields", rather than "after the last compositional field has been updated"?

Reply:
Oh, or are you thinking of a multistep RK scheme? In any case, I think that would be worth outlining.